packages/d3fc-annotation/examples/line.js

```diff
@@ -1,24 +1,42 @@
 /* global xScale, yScale, render */
-var horizontalLine = fc.annotationSvgLine()
+var horizontalSvgLine = fc.annotationSvgLine()
   .label('')
   .xScale(xScale)
   .yScale(yScale);
 
-var verticalLine = fc.annotationSvgLine()
+var verticalSvgLine = fc.annotationSvgLine()
   .orient('vertical')
   .label('')
   .xScale(xScale)
   .yScale(yScale);
 
+var horizontalCanvasLine = fc.annotationCanvasLine()
+  .label('')
```

Comment:
As we're not showing the axis, perhaps we could display the label for the `canvas` version. That would also allow you to demonstrate using different fill/stroke colours form the line and text.